### DISCUSSION TITLE
Adding Payment - SEPA

### DISCUSSION THREAD

**Discussion by NiklasvG** [swellstores/community#162] [Help] [answered]
I am creating a shop for a German customer. This customer would like to use the SEPA procedure as a payment option. Is this somehow possible with Swell?

I added Stripe, but I can only select a few providers in the advanced options and not all I added at Stripe directly. 

If I try to add PayPal, it opens just a window to let me add a sandbox account and not my normal business account.

**Comment by greph** [comment, marked as answer]
@NiklasvG Swell currently doesn't support SEPA out of the box. However, a custom payment method could be configured via Stripe elements and setup as a custom payment method on Swell. 
https://stripe.com/docs/payments/sepa-debit/accept-a-payment?platform=web&ui=element
